tools-local/tasks/mobile.tasks/WasmAppBuilder/WasmAssemblySearcher.cs

```diff
@@ -0,0 +1,159 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Reflection;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+public class WasmAssemblySearcher : Task
```

Comment:
IMHO, the name doesn't indicate what this task really does. `WasmGetReferencedAssemblies`? or something better

Reply:
How does `WasmLoadAssembliesAndReferences` sound?